packages/documentation/docs/migration/2_0_0.md

```diff
@@ -0,0 +1,40 @@
+---
+sidebar_position: 0
+sidebar_title: Migration to 2.0.0
+title: Migration to 2.0.0
+---
+
+# ix-menu
+
+## Item dropdown overflow is removed
+
+We replaced the overflow menu with second level navigation in form of categories
```

Comment:
```suggestion
## More items dropdown was removed

With the introduction of a second level navigation menu items can now be nested to save screen real estate.
In case the menu grows past the available space it will scroll vertically.
```